test/helpers.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+var expect = require('must');
+var STANDARD_DELAY = 200; // 200ms between nested calls to the API so things settle
+
+function assertSuccessful(done, cb) {
```

Comment:
Is the aim of these helpers to replace only those tests that have those assertions? Stated in other words, if a callback for a test has:

```
expect(err).not.to.exist();
expect(res).to.exist();
expect(xhr).to.be.an.object();
```

I should use this helper, but if the test has more I should not use it?